{"name": "test_3", "status": "skipped", "statusDetails": {"message": "Skipped: there is no reason, give up!!", "trace": "('F:\\\\jenkins_test\\\\test.py', 16, 'Skipped: there is no reason, give up!!')"}, "start": 1601967013699, "stop": 1601967013699, "uuid": "2726c875-0df3-420e-9620-06b2f4bb82f1", "historyId": "1f79a7926d2bb1598ac0f050be41a1b4", "testCaseId": "232ed1e320fdb0fae2684fcf80b8780d", "fullName": "test#test_3", "labels": [{"name": "tag", "value": "@pytest.mark.skip(reason='there is no reason, give up!!')"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "DESKTOP-92UGMTH"}, {"name": "thread", "value": "8636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test"}]}